outputs.tf

```diff
@@ -5,8 +5,32 @@
 output "security_group" {
   description = "The security group attributes."
   value = try(
-    aws_security_group.self[0],
```

Comment:
Outputting all sg attributes, including `ingress` and `egress` attributes, will result in changes with the `aws_secruity_group` resource when associated `aws_security_group_rule`(s) are updated.